test/js/specs/happy.js

```diff
@@ -1,16 +1,48 @@
 const path = require('path')
+const expect = require('chai').expect;
 import {describe, it} from '../utils/mochaw'
 
 const options = {
   pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
 }
 
-describe('Happy Paths',options, ({driver,pageObjects}) => {
+const localhostUrl = 'https://localhost:8080/'
+
+describe('Happy Paths',options, ({driver,$,pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
+  //Welcome Screen Tests
+  it('test website title', async () => {
+    await driver.get(localhostUrl)
+    const title = await driver.getTitle();
+    expect(title).to.equal('Onfido SDK Demo');
+  })
+
+  it('test welcome screen title', async () => {
+    const welcomeTitleText = await welcome.welcomeTitle.getText()
+    expect(welcomeTitleText).to.equal('Open your new bank account');
+    await welcome.welcomeTitle.isDisplayed()
+  })
+
+  it('test welcome screen subtitle', async () => {
+    const welcomeSubtitleText = await welcome.welcomeSubtitle.getText()
+    expect(welcomeSubtitleText).to.equal('To open a bank account, we will need to verify your identity.' + '\n' + 'It will only take a couple of minutes.');
+    await welcome.welcomeSubtitle.isDisplayed()
+  })
+
+  it('test verify identity button', async () => {
+    const verifyIdentityBtnText = await welcome.primaryBtn.getText()
+    expect(verifyIdentityBtnText).to.equal('Verify Identity');
```

Comment:
We will have to use keys from our local files here. Is this going to be handled as part of another story?

Reply:
I didn't consider it as part of this ticket. I'm not entirely sure how I could do that if it's a quick win I'm happy to do that as this would be much better to actually use the keys rather just strings.

Reply:
I guess you can add a function to fetch the key. Something like @rfreitas did in [this PR](https://github.com/onfido/onfido-sdk-ui/pull/620/files) 
```
copy = (lang="en") =>
      require(`../../../src/locales/${lang}.json`)["welcome"]["next_button"]
```

Reply:
This function should be put inside `test/js/pageobjects/Welcome.js`